### ISSUE TITLE
Cannot create interface on Ubuntu 16.04

### ISSUE THREAD

**Issue by esellin** [target/network_interfaces_v2-cookbook#25]
```
network_interface 'eth0:0' do
  bootproto 'static'
  address fo_server_ip
  netmask '255.255.255.255'
  broadcast fo_server_ip
end
```

`service[modules-load] (modules::config line 56) had an error: Errno::ENOENT: No such file or directory - /sbin/status`

This is most likely due to Ubuntu moving from Upstart to SystemD.

**Comment by esellin** [comment]
Maybe the fix is similar to this?

[Handle SystemD on ubuntu 15.04+](https://github.com/chef-cookbooks/openssh/pull/62/commits/ee32e90bc66d94a22b0ff82814e49d8d1b3d931e)

**Comment by jmccann** [comment]
Thanks for the issue ... will look into it!

**Comment by jmccann** [comment]
It seems the issues stems from the `modules` cookbook ... which hasn't been updated since mid 2015.  I'll need to either find a newer cookbook to do the needful or write up some code myself.

```
             ================================================================================
             Error executing action `enable` on resource 'service[modules-load]'
             ================================================================================
             
             Errno::ENOENT
             -------------
             No such file or directory - /sbin/status
             
             Resource Declaration:
             ---------------------
             # In /tmp/kitchen/cache/cookbooks/modules/recipes/config.rb
             
       56:   service "modules-load" do
       57:     provider Chef::Provider::Service::Upstart
       58:     action [:enable, :start]
       59:     notifies :start, "service[module-init-tools]"
       60:   end
       61: else
```

**Comment by jmccann** [comment]
This should be addressed by 6cc846846841eecd7f86a8e9931c28ff78e19057 and will be released shortly.

**Comment by jmccann** [comment]
Just released `2.9.0` with this support.
